local/channel-rust-stable.toml

```diff
@@ -2,4 +2,4 @@
 # promote-release runs, to force it to do a new release.
 
 [pkg.rust]
-version = "0.0.0-dummy (000000000 1970-01-01)"
+version = "0.0.0 (000000000 1970-01-01)"
```

Comment:
Hm, could we instead just handle 404s/missing files appropriately (as needing a new manifest)?

(Probably out of scope for this PR)

Reply:
I guess. I'll followup with another PR.